src/build/validator.md

```diff
@@ -49,12 +89,16 @@ sudo docker-compose --env-file full-node.client.testnet.public01.arg -f init-ful
 
 If all steps are completed successfully, you should see a newly generated folder called **CudosData** at the same directory where you placed *CudosBuilders* and *CudosNode*. The subdirectory *cudos-data-full-node-client-testnet-public-01* of **CudosData"** folder has a file called **tendermint.nodeid**. This file contains your node **Id,** to see your node id you can open this file in any code editor and you will get one line that represents your node id such as 13f359c90582b12e291311980a855854668d80pc.
 
-#### Configure and start the node
+#### Configure and start the Full node
 
 This step is valid only if you are running the full node as a validator. Note that if you are not a validator, you do not need to follow this step.
```

Comment:
will we be directing developers who want to run a full node to these validator instructions? Should we leave this page strictly for validator instructions, and create a new one for developers to run a full-node, with only the relevant commands and configs?

Reply:
agreed, we need a Devs section within /build/

Reply:
Is there a big difference? Which steps will be different for developers? they do not need to run Ethereum node and do not need to set up the Gravity bridge?

Reply:
it's a very different user journey so we'll be directing towards 1) full node setup, 2) smart contracts, 3) dev tools

this is all currently missing from the docs so needs completing in a new PR i think

Reply:
agreed, I would suggest removing the references to instructions that are not for validators in this PR though, and then we open a new one (or use Aditya's) for all the developer content

Reply:
Noted that for next article to be for develoeprs